plaidml/CMakeLists.txt

```diff
@@ -26,6 +26,24 @@ pml_cc_library(
     pmlc::rt::all_runtimes
     pmlc::target::all_obj
 )
+else()
+pml_cc_library(
+  NAME plaidml
+  TYPE STATIC
+  LINKOPTS
+  PROPS
+    LINK_DEPENDS ${WINDOWS_DEF}
```

Comment:
Not sure we need a `.def` file if we're just going to statically compile. Also, the intended use case for this library is for python, and so it pretty much has to be a `MODULE` or `SHARED` type.

Reply:
I got a problem with this one. As I am using this for plaidmlPlugin for OpenVino I believe it should be statically linked. I did not see other solutions for different plugins. I was thinking on building both static and shared libs, so the shared one can be used in python and static for the c++ content.

Reply:
That makes sense to me (having dual static/shared).